sandbox/cgroup/freezer.go

```diff
@@ -178,10 +188,48 @@ func writeExistingFile(where string, data []byte, mode os.FileMode) error {
 	return errC
 }
 
+var freezeOneV2 = func(ctx context.Context, dir string) error {
+	groupName := filepath.Base(dir)
+	fname := filepath.Join(dir, "cgroup.freeze")
+	if err := writeExistingFile(fname, []byte("1")); err != nil {
+		if os.IsNotExist(err) {
+			//  the group may be gone already
+			return nil
+		}
+		return fmt.Errorf("cannot freeze processes in group %q: %v", groupName, err)
```

Comment:
I think we can preserve the original error here.

```suggestion
		return fmt.Errorf("cannot freeze processes in group %q: %w", groupName, err)
```